src/com/aca/homework/week6/list/ArrayList.java

```diff
@@ -0,0 +1,45 @@
+package com.aca.homework.week6.list;
+
+public class ArrayList implements List {
+    private String[] values = new String[10];
+    private int length = 0;
+
+    public void add(String value) {
+        if (isArrayListFull()) {
+            assignToDoubleSizedArray();
+        }
+        values[length++] = value;
+    }
+
+    public String get(int index) {
```

Comment:
if the user adds null he/she can not get that

Reply:
have just modified